src/lib/config/index.ts

```diff
@@ -54,15 +55,15 @@ export interface Config {
   >;
 }
 
-const config: Record<string, Config> = {
-  '1': homestead,
-  '42': kovan,
-  '4': rinkeby,
-  '137': polygon,
-  '42161': arbitrum,
-  '12345': test,
+const config: Record<NetworkId | 12345 | 17, Config> = {
+  [Network.MAINNET]: homestead,
+  [Network.KOVAN]: kovan,
+  [Network.RINKEBY]: rinkeby,
+  [Network.POLYGON]: polygon,
+  [Network.ARBITRUM]: arbitrum,
+  12345: test,
   // @ts-ignore
-  '17': docker
+  17: docker
```

Comment:
Is there maybe a good reason to use strings instead of numbers as the key?

Reply:
Just a preference I think... just gotta pick one and stick with it. (its going to be easier once everything uses the `enum`).

The docker/test are special cases which I don't think need to be included in that `enum`.